pkgs/tools/misc/farge/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bash
+, bc
+, feh
+, grim
+, imagemagick
+, slurp
+, wl-clipboard
+, xcolor
+, waylandSupport ? true
+, x11Support ? true
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "farge";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "sdushantha";
+    repo = "farge";
+    rev = "v" + finalAttrs.version;
+    hash = "sha256-rTY4vQkrcyVRfIHt1qtFpxaC5bmpJ2/C6NVhEDkxef8=";
+  };
+
+  buildInputs = [ bash bc feh ]
+    ++ lib.optionals waylandSupport [ grim slurp wl-clipboard ]
+    ++ lib.optionals x11Support [ xcolor ];
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
I think (please verify if I'm wrong), that you can get rid of mkdir, the next line will create it.

Reply:
Ah that's interesting, thanks :+1: